etc/behat/services.xml

```diff
@@ -30,6 +30,7 @@
         <parameter key="sylius.behat.page.admin.crud.create.class">Sylius\Behat\Page\Admin\Crud\CreatePage</parameter>
         <parameter key="sylius.behat.page.admin.crud.update.class">Sylius\Behat\Page\Admin\Crud\UpdatePage</parameter>
         <parameter key="sylius.behat.service.accessor.notification_accessor.class">Sylius\Behat\Service\Accessor\NotificationAccessor</parameter>
+        <parameter key="sylius.behat.service.notification_validator.class">Sylius\Behat\Service\NotificationValidator</parameter>
```

Comment:
Perhaps we should be consistent and name these:

``sylius.behat.service.accessor.notification``
``sylius.behat.service.validator.notification``

?

Or maybe name this... Asserter!!! :smiling_imp: